post-processing/post_processing.py

```diff
@@ -111,6 +120,61 @@ def run_post_processing(self, log_path, config):
 
         return df[columns][mask]
 
+    def plot_generic(self, title, df, x_axis, y_axis, series_filters):
+        """
+            Create a bar chart for the supplied data using bokeh.
+
+            Args:
+                title: str, plot title (read from config).
+                df: dataframe, data to plot.
+                x_axis: dict, x-axis column and units (read from config).
+                y_axis: dict, y-axis column and units (read from config).
+                series_filters: list, x-axis groups used to filter graph data.
+        """
+
+        # get column names of axes
+        x_column = x_axis.get("value")
+        y_column = y_axis.get("value")
+        # get units
+        x_units = df[x_axis["units"]["column"]][0] if x_axis.get("units").get("column") \
+                  else x_axis.get("units").get("custom")
+        y_units = df[y_axis["units"]["column"]][0] if y_axis.get("units").get("column") \
+                  else y_axis.get("units").get("custom")
+        # determine axis labels
+        x_label = "{0}{1}".format(x_column.replace("_", " ").title(),
+                                  " ({0})".format(x_units) if x_units else "")

```

Comment:
Should other group column names also be added to the x-axis label?

Reply:
Yes, but I'll move this to a new issue, since we have to fix the 2-level grouping anyway.

Reply:
See #198